### ISSUE TITLE
[Suggestion] Recycling/Reroll/uses for duplicates ala Nameless Trinkets or Vault Hunters etc.

### ISSUE THREAD

**Issue by Baggytrousers27** [ochotonida/artifacts#180]
Got 5 pairs of Kitty slippers, 3 umbrellas, bunny hoppers coming out of my ears and enough whoopee cushions to sink (or refloat) a battleship.
Doesn't even need a reroll mechanic, just something to do with all my spares (Running out of armour stands).

**Comment by ochotonida** [comment]
I'm still not really sure if this is a problem that needs to be addressed, since could argue that the same issue applies to vanilla items you can only find in chests like saddles and horse armor. A rerolling mechanic in particular is something I'm not interested in adding, as I think this would discourage players from exploring in different places.
I'll keep this issue open for now though.

**Comment by Aceplante** [comment]
I agree being able to just reroll would disincentivize exploring. I think a better solution would be to be able to break them down into artifact dust or something, which can have some other use in leveling up your other artifacts? That would make collecting duplicate artifacts always worth it, and give an easier way to level up some of the ones that are harder to or require insane grinding (rooted boots for example)
